src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -8,8 +8,168 @@ authors:
   - Ysobel Sims (@ysims)
   - Josephus Paye II (@JosephusPaye)
   - Lachlan Court (@LachlanCourt)
+  - Clayton Carlon (@Claegtun)
 ---
 
+## 2025: Salvador, Brazil
+
+### What went wrong
+
+#### Hardware
+
+- Necks broke often; screws would sometimes come out of the horn.
+- Frankie would brown out after falling.
+- Sarah's torso-pitch is off despite correct offsets.
```

Comment:
```suggestion
- Sarah's torso pitch is significantly off during motions despite correct zero position.
```
Maybe just to clarify what this means. The offsets aren't necessarily correct, just that the zero position looks right.

Reply:
Done.